src/components/utils/validation.js

```diff
@@ -88,7 +88,7 @@ export function isVmNameValid (nameCandidate: string): boolean {
 }
 
 export function isDiskNameValid (nameCandidate: string): boolean {
-  return isVmNameValid(nameCandidate)
+  return /^[a-zA-Z0-9_\-\\.]{1,40}$/.test(nameCandidate)
```

Comment:
Please update  this length validation "{1,40}" to a much longer length (e.g. 64 chars)

Reply:
@sgratch Are you sure?
I stuck to the [documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/administration_guide/sect-virtual_disk_tasks#Creating_a_Virtual_Disk)

Reply:
> @sgratch Are you sure?
> I stuck to the [documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/administration_guide/sect-virtual_disk_tasks#Creating_a_Virtual_Disk)

When comparing webadmin code/behaviour to the documentation, usually the webadmin behaviour is the the enforcing rule (and probably the doc is not up to date) :). 
Specifically here, the limit is 255 chars:
https://github.com/oVirt/ovirt-engine/blob/e48484ccb3c1c8fcaed81023b234295a109ebd8b/frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/DisksAllocationModel.java#L486-L490

Reply:
OK, I'll check webAdmin code next time :)